src/util/PinConfig.js

```diff
@@ -1,17 +1,26 @@
-import Const from './PinConst';
+import { COUNT_TYPES } from './PinConst';
 
-let guid = '';
-for (var i = 0; i < 12; i++) {
-    guid += Const.GUID_VARS.substr(Math.floor(Math.random() * 60), 1);
+/**
+ * Generates a random pseudo-GUID
+ * @return {string} GUID
+ */
+function generateGuid() {
+    const GUID_VARS = '0123456789ABCDEFGHJKLMNPQRSTUVWXYZ_abcdefghijkmnopqrstuvwxyz';
+    let guid = '';
+    for (var i = 0; i < 12; i++) {
+        guid += GUID_VARS.substr(Math.floor(Math.random() * 60), 1);
+    }
+    return guid;
 }
 
-export default {
-    guid: guid,
-    [Const.COUNT.BUTTON]: 0,
-    [Const.COUNT.FOLLOW]: 0,
-    [Const.COUNT.PIN_SMALL]: 0,
-    [Const.COUNT.PIN_MEDIUM]: 0,
-    [Const.COUNT.PIN_LARGE]: 0,
-    [Const.COUNT.PROFILE]: 0,
-    [Const.COUNT.BOARD]: 0
-};
\ No newline at end of file
+export const guid = generateGuid();
```

Comment:
Not quite sure why we've got this GUID? I think this is generated once per run, so may move into Constants